cypress/integration/networking.spec.ts

```diff
@@ -0,0 +1,65 @@
+import type { OnboardedNode } from "../plugins/index";
+
+import * as commands from "../support/commands";
+import * as path from "path";
+import { ipcStub } from "../support";
+
+context("networking", () => {
+  it("replicates a project from one node to another", () => {
+    cy.task("startNode", 17000);
```

Comment:
I wonder if we could do something similar to `radicle-link`s approach where we can specify the number of nodes to boot up and get back a list of them. I'd also wonder if we should randomly generate the set of `NodeId`s and maybe probe to see if that port is taken yet.

Reply:
That was my initial approach, I wanted to start the proxy listening on random ports via `radicle-proxy --http-listen 127.0.0.1:0 --peer-listen 0.0.0.0:0` and then read out the ports that the OS picked from the logs on STDOUT/STDERR. But I got stuck when I couldn't figure out a way to read out the actual port that `warp` bound to.

So I opted for specifying the ports manually as a first step to move forward with the implementation.
I'd love to work on this as a follow up, but will need some help figuring out how to get the bound ports.

Reply:
Ya, I was thinking more like:
```
getRandomPort() => number {
  let available = false;
  let port = random(9000, 10_000);
  while !available { 
    available = test_port(port);
  }
}
```

Where there's some utility for checking a port is taken by the OS. But definitely makes sense to do it in a follow-up

Reply:
Yea, that could also work. I just wanted to avoid writing the utility and eliminate a possible race condition. Will keep this in mind for the follow-up. Marking this as resolved for now.